website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -69,6 +69,9 @@ for a subset of older versions and deprecate the others.
 
 # Released Upgrade Notes
 
+## Upgrading to v0.12.1+
```

Comment:
I'm not sure if this should be a patch. I guess it's technically backwards compatible and it doesn't really introduce a new feature, but it will cause errors in the logs.

Reply:
SGTM